Source/DafnyLanguageServer.Test/Lookup/HoverVerificationTest.cs

```diff
@@ -229,6 +242,9 @@ method Abs(x: int) returns (y: int)
       await client.OpenDocumentAndWaitAsync(documentItem, CancellationToken);
       var lastStatus = await WaitUntilDafnyFinishes(documentItem);
       Assert.AreEqual(expectedStatus, lastStatus);
+      foreach (var document in Documents.Documents) {
+        await document.LastDocument;
```

Comment:
@keyboardDrummer  do your confirm it's the right way to wait for the final translated and verified document to be available?

Reply:
Great, that's what I wanted to know.